src/components/book.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import './book.css';
+
+function Book(props) {
```

Comment:
[OPTIONAL]

You can also destructure your props directly inside the function parameters
e.g
`function Book({ title, author } ) {...}`